src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/08.md

```diff
@@ -3,48 +3,48 @@
+In 15th century Bavaria a fashionable beer style emerged: white (*weisse*) beer was lighter than Bavarian lagers because of some brewer manipulations. One of the means of ‘whitewashing’ beer is using wheat (*Weizen*): wheat beer gets a high white foamy head, and wheat itself may be used fully or partly unmalted. By approximately the end of the 15th century these two words — *Weissbier* (e.g. ‘white beer’) and *Hefeweizen* (literally, ‘yeast wheat’) — started to mean exactly the same thing, wheat beer.
 
-There is no consensus among scholars whether using wheat for brewing was a real food security problem, but the fact is undeniable: by introducing the ‘Beer Purity Law’, Wilhelm IV had legally outlawed using any grain but barley. However, Wilhelm's son, Albrecht V, made an exception to this rule in a form of a state monopoly — presumably under the pressure of consumers that were not willing to comply. Initially, the right to brew wheat beer had been granted to Count Degenburg, but after his death returned back to the Crown. The reigning Duke, Maximilian I, had quickly learned that this monopoly might have brought considerable profits as Bavarians liked wheat bear much and were eager to pay for the ‘forbidden fruit’, especially as it was the only kind of beer allowed to be brewed in summer. Maximilian founded an entire network of state breweries, one of which — former Weisses Bräuhaus — has survived until today[ref:{"short":"Staudt, A. (2015)","extra":["Brewing Bavarian Weissbier – all you ever wanted to know","brau!magazin"]}](https://braumagazin.de/article/brewing-bavarian-weissbier-all-you-ever-wanted-to-know/).
+There is no consensus among scholars whether using wheat for brewing was a real food security problem, but one fact is indisputable: by introducing the ‘Beer Purity Law’, Wilhelm IV had legally outlawed using any grain but barley. However, Wilhelm's son, Albrecht V, made an exception to this rule in a form of a state monopoly — presumably under consumer pressure of those unwilling to comply. Initially, the right to brew wheat beer had been granted to Count Degenburg, but after his death this brewing *charter* reverted back to the Crown. The reigning Duke, Maximilian I, had quickly learned that this monopoly might bring considerable profits as Bavarians liked wheat bear immensely and were eager to pay for the ‘forbidden fruit’, especially as it was the only kind of beer allowed to be brewed in summer. Maximilian founded an entire network of state breweries, one of which — the former Weisses Bräuhaus — has survived until today[ref:{"short":"Staudt, A. (2015)","extra":["Brewing Bavarian Weissbier – all you ever wanted to know","brau!magazin"]}](https://braumagazin.de/article/brewing-bavarian-weissbier-all-you-ever-wanted-to-know/).
```

Comment:
@JoRembert I've reverted 'the only beer allowed in summer heat' to 'allowed to be brewed in summer'. It was not banned to consume beer — if you had it stored since winter, which was of course expensive. It was not allowed to brew new batches of beer.

Reply:
Clear